src/app/problems/problem-view/problem-view.component.html

```diff
@@ -76,6 +76,7 @@
         <ng-container *ngIf="!!uqj">
             <app-submission-snippet
                 [questionId]="uqj.question.id"
+                [inputFiles]="uqj.input_files"
```

Comment:
this is passing over all the input_files because I could not figure out a way to access and retain gathered information from uqjs in any other way